script/build-client.bash

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+if [ -d "./node_modules/@openstax/highlights-client" ]; then
+  cp -r "./node_modules/@openstax/highlights-client" .
+  exit 0
+fi
+
+if [ -d "./highlights-client" ]; then
+  exit 0
+fi
+
```

Comment:
should there be some check to make sure the `highlights-client` directory built successfully?  Otherwise it might error in the build phases below and then seem to succeed if you re-run it